js/init.js

```diff
@@ -1,8 +1,17 @@
+var markitupInit = {
+	init: function() {
+		$('.markitup:not(.markItUpEditor)').markItUp(myTextileSettings);
+
+		$('.markItUpButton1 a').html('<i class="icon-link"></i>');
+		$('.markItUpButton2 a').html('<i class="icon-italic"></i>');
+		$('.markItUpButton3 a').html('<i class="icon-bold"></i>');
+		$('.markItUpButton4 a').html('<i class="icon-strikethrough"></i>');
+	}
+}
 $(document).ready(function() {
-	$('textarea.markitup').markItUp(myTextileSettings);
+	markitupInit.init();
+});
 
-	$('.markItUpButton1 a').html('<i class="icon-link"></i>');
-	$('.markItUpButton2 a').html('<i class="icon-italic"></i>');
-	$('.markItUpButton3 a').html('<i class="icon-bold"></i>');
-	$('.markItUpButton4 a').html('<i class="icon-strikethrough"></i>');
+$('#sections-sortable').on('new-section.pagemanager', function() {
+	markitupInit.init();
```

Comment:
I'd be better having this in https://github.com/propcom/fuel-cms-assets. Classes can be renamed, and is easy to remember what packages use it.
Or use ```markitupInit.init()``` in https://github.com/propcom/fuel-cms-assets, as it is global variable.